src/dguweb/web/templates/user/index.html.eex

```diff
@@ -0,0 +1,15 @@
+
+<h1 class="heading-medium">My publishers</h1>
```

Comment:
Elsewhere I put heading-xlarge. Can you change so they're all the same? I slightly lean towards medium or large